app/assets/javascripts/admin_users.js

```diff
@@ -43,10 +43,15 @@
 
   // Hide password fields when non-internal authentication source is selected
   function on_auth_source_change() {
+    var passwordFields = jQuery('#password_fields');
+        passwordInputs = passwordFields.find('#user_password, #user_password_confirmation');
```

Comment:
well spotted @houndci!

@ulferts The line before ends in a `;`, which makes this line define the variable globally instead of locally...

Either add another `var` prefix or replace the semicolon in the line before with a `,`